### ISSUE TITLE
Adding "best practices" section

### ISSUE THREAD

**Issue by azarboon** [adr/adr.github.io#55]
Hi. I'm thinking to add a "best practices" section. This way readers can get list of best practices in the main page without the hassle of going through several articles. How does it sound? If maintainers support this idea, I can start it. @koppor

**Comment by socadk** [comment]
Thanks for the suggestion @azarboon. Can you describe your proposal in some more detail please: do you think of a new page? how would its content be curated/quality-assured?

**Comment by azarboon** [comment]
> Thanks for the suggestion @azarboon. Can you describe your proposal in some more detail please: do you think of a new page? how would its content be curated/quality-assured?

Thank you. I was thinking about a new section in the main page titled as "best practices". But we can create a new page and link it in the main page.

Content will be based on existing articles and some other blogs. I can initiate the first draft and others can improve it. Once the maintainers endorse it, it should be good enough. And as always, we keep it open for further improvements.

Please let me know your advice.

**Comment by azarboon** [comment]
@koppor what's your advice?

**Comment by koppor** [comment]
Sure, go ahead!

**Comment by socadk** [comment]
@koppor @azarboon I thought about the proposal a bit more... I am not sure whether a "one size fits all" page with agreed upon good or best practices is feasible to create and maintain: Contexts and cultures differ, preferences and requirements differ. I suggest to move all pointers to good or best practices to a new page, possibly pointing at selected highlights and a disclaimer that we are merely _pointing_ at advice, not _endorsing_ it.

**Comment by azarboon** [comment]
> @koppor @azarboon I thought about the proposal a bit more... I am not sure whether a "one size fits all" page with agreed upon good or best practices is feasible to create and maintain: Contexts and cultures differ, preferences and requirements differ. I suggest to move all pointers to good or best practices to a new page, possibly pointing at selected highlights and a disclaimer that we are merely _pointing_ at advice, not _endorsing_ it.

Sounds good.

**Comment by koppor** [comment]
We might need a scope clarification. Maybe, "decision making" is too borad for this repo. We might provide some initial pointers (e.g., https://medium.com/olzzio/a-definition-of-ready-for-architectural-decisions-ads-2814e399b09b), but cannot claim "completeness" in this field. "decision caputring" is more in the scope of this repository.

**Comment by azarboon** [comment]
> We might need a scope clarification. Maybe, "decision making" is too borad for this repo. We might provide some initial pointers (e.g., https://medium.com/olzzio/a-definition-of-ready-for-architectural-decisions-ads-2814e399b09b), but cannot claim "completeness" in this field. "decision caputring" is more in the scope of this repository.

Interesting. I have to go through that link in detail. However, let's not get stuck in analysis paralysis. I can make the first draft and we can refine it and reiterate.

Before working on the draft, please finalize my other open PR. Then we can proceed with this one.

**Comment by koppor** [comment]
> Interesting. I have to go through that link in detail. However, let's not get stuck in analysis paralysis. I can make the first draft and we can refine it and reiterate.

To provide room for expirments, we changed the layout of the homepage (https://github.com/adr/adr.github.io/pull/58) and currently restructuring into separate sections. Shouldn't take long.

**Comment by azarboon** [comment]
@koppor can you please create a new page named "best practices" so we can start to work on this. Also please add "edit this page" to the new layout.

**Comment by koppor** [comment]
> Also please add "edit this page" to the new layout.

Filed as https://github.com/adr/adr.github.io/issues/68

I think, we need to implement for ourselves as soon as we find time for that.

**Comment by koppor** [comment]
@azarboon Note that we do not want to add "realy" content to the page; "just" links. (See https://github.com/adr/adr.github.io/issues/55#issuecomment-2435465716). Thus, if you have in mind to describe a concrete "good" practicse for ADRs, please craft the text on another site and we can link it when resolving this usse.

**Comment by azarboon** [comment]
> @azarboon Note that we do not want to add "realy" content to the page; "just" links. (See [#55 (comment)](https://github.com/adr/adr.github.io/issues/55#issuecomment-2435465716)). Thus, if you have in mind to describe a concrete "good" practicse for ADRs, please craft the text on another site and we can link it when resolving this usse.

I was thinking about a concise summary of best practices (a long with links for further) to save readers' time.

**Comment by koppor** [comment]
> I was thinking about a concise summary of best practices (a long with links for further) to save readers' time.

Please have an eye on a sound order of your summary entries and high cohesion between lists. -- How about adding the list at https://adr.github.io/ad-practices/#third-party-articles ?

Looking ahead, it might make sense to start another GitHub repo/page for this. Please note that we do not want to endorse any particular practices here; we are reluctant to use the term "best practices", maybe "good practices" is a little more fitting?

**Comment by azarboon** [comment]
> > I was thinking about a concise summary of best practices (a long with links for further) to save readers' time.
> 
> Please have an eye on a sound order of your summary entries and high cohesion between lists. -- How about adding the list at https://adr.github.io/ad-practices/#third-party-articles ?
> 
> Looking ahead, it might make sense to start another GitHub repo/page for this. Please note that we do not want to endorse any particular practices here; we are reluctant to use the term "best practices", maybe "good practices" is a little more fitting?

Yes, indeed I was thinking to extract good practices from different articles and list them and add link for further reading.  Also, some practices are shared among different articles. If something is debatable or nuanced, we should clarify it.

At its core, this is going to be "best practices" or "good practices" and definitely opinionated. However, I would argue it's highly beneficial. Ofcourse we can attach a disclaimer.

Given how cautious you are about not endorsing any of those tips, I think it's better to create a separate page for it but within the same repo (I guess?). Please let me know your advice. @koppor
